portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -778,6 +778,71 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
+	@description = " Verify that a user on a form that limits one submission per user can see partial results when accessing the forms link."
```

Comment:
Hey @thaiscabrals!
Could you add the related story LPS? This is something we can discuss with the team to start doing (when it's possible)